.gitignore

```
@@ -12,7 +12,7 @@
 !Makefile
 !*.cpp
 !*.hpp
-!test/**
+!*.py
 
```

これ欲しいと思ってました！
test ディレクトリの中の `.py` だけで良さそうですかね？(細かくてすみません)
```suggestion
!test/*.py
```